script.js

```diff
@@ -0,0 +1,432 @@
+    const notes = document.createElement('div');
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+
+    // Setup main elements
+    this.elements.input.classList.add('input-field');
+    notes.innerHTML = this.notesInnerHTML;
+    notes.classList.add('notes-container');
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard__keys');
+
+    // Put elements into DOM
+    document.body.appendChild(this.elements.input);
+    document.body.appendChild(notes);
+    this.elements.main.appendChild(this.elements.keysContainer);
+    document.body.appendChild(this.elements.main);
+
+    keyboard.createKeys();
+  },
+
+  capsLockFunc() {
+    if (keyboard.shift === false && keyboard.lang === 'en') {
+      const spanz = document.querySelectorAll('.keySpan');
+      const buttonz = document.querySelectorAll('.SHIFT');
+      for (let i = 0; i < spanz.length; i += 1) {
+        spanz[i].classList.add('HIDDEN');
+      }
+      for (let i = 0; i < buttonz.length; i += 1) {
+        buttonz[i].classList.remove('HIDDEN');
+      }
+      keyboard.shift = true;
+      localStorage.setItem('virtualKeyboardShift', true);
+    } else if (keyboard.shift === true && keyboard.lang === 'en') {
+      const spanz = document.querySelectorAll('.keySpan');
+      const buttonz = document.querySelectorAll('.ENG');
+      for (let i = 0; i < spanz.length; i += 1) {
+        spanz[i].classList.add('HIDDEN');
+      }
+      for (let i = 0; i < buttonz.length; i += 1) {
+        buttonz[i].classList.remove('HIDDEN');
+      }
+      keyboard.shift = false;
+      localStorage.setItem('virtualKeyboardShift', false);
+    } else if (keyboard.shift === false && keyboard.lang === 'rus') {
+      const spanz = document.querySelectorAll('.keySpan');
+      const buttonz = document.querySelectorAll('.RUS_SHIFT');
+      for (let i = 0; i < spanz.length; i += 1) {
+        spanz[i].classList.add('HIDDEN');
+      }
+      for (let i = 0; i < buttonz.length; i += 1) {
+        buttonz[i].classList.remove('HIDDEN');
+      }
+      keyboard.shift = true;
+      localStorage.setItem('virtualKeyboardShift', true);
+    } else if (keyboard.shift === true && keyboard.lang === 'rus') {
+      const spanz = document.querySelectorAll('.keySpan');
+      const buttonz = document.querySelectorAll('.RUS');
+      for (let i = 0; i < spanz.length; i += 1) {
+        spanz[i].classList.add('HIDDEN');
+      }
+      for (let i = 0; i < buttonz.length; i += 1) {
+        buttonz[i].classList.remove('HIDDEN');
+      }
+      keyboard.shift = false;
+      localStorage.setItem('virtualKeyboardShift', false);
+    }
+  },
+
+  triggerMouseEvent(node, eventType) {
+    const clickEvent = document.createEvent('MouseEvents');
+    clickEvent.initEvent(eventType, true, true);
+    node.dispatchEvent(clickEvent);
+  },
+
+  shiftBtnPressed: false,
```

Comment:
Because we don't want to pollute the global namespace, right?

The property 'shiftBtnPressed' is used here:

window.addEventListener('customEvent', () => {
  if (keyboard.controlBtnPressed && keyboard.shiftBtnPressed) {
    keyboard.changeLanguage();
  }
});